update.sh

```diff
@@ -24,10 +24,37 @@ declare -A extras=(
 	[fpm-alpine]=''
 )
 
+apcu_version="$(
+	git ls-remote --tags https://github.com/krakjoe/apcu.git \
```

Comment:
Are you sure that the version is available on pecl as soon as it's tagged on github?

Reply:
Nope, but it's hard to fetch the version from PECL.

Reply:
have you tried using the rss feed? https://pecl.php.net/feeds/pkg_APCu.rss

Reply:
I tried to use xmllint, but it's still harder than using git tags.

Reply:
I think git tags a fine for now

if a lot of builds end up failing, we can still change it to something hacky like this:
```bash
curl -fsSL https://pecl.php.net/feeds/pkg_APCu.rss |tac|tac| grep -oE '<title>APCu [[:digit:]]+(\.[[:digit:]]+){2}</title>' | grep -oE '[[:digit:]]+(\.[[:digit:]]+){2}'
```